portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,39 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+header {
+    text-align: center;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#home-body {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 50%;
 }
+
+img {
+    height: 500px;
+    width: 500px;
+}
+
+.topnav {
+  overflow: hidden;
+  background-color: #f1f1f1;
+}
+
+.topnav a {
+  float: left;
+  display: block;
+  color: black;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+  border-bottom: 3px solid transparent;
+}
+
+.topnav a:hover {
+  border-bottom: 3px solid red;
+}
+
+.topnav a.active {
+  border-bottom: 3px solid red;
+}
```

Comment:
Add newline to end of files, plz!

Reply:
I've been updating my CSS file with more things, but I'll add this in my next commit